kp.go

```diff
@@ -256,6 +257,12 @@ func (c *Client) do(ctx context.Context, req *http.Request, res interface{}) (*h
 		req.Header.Set("x-kms-key-ring", c.Config.KeyRing)
 	}
 
+	if c.Config.Headers != nil {
+		for key, value := range c.Config.Headers {
+			req.Header.Set(key, value[0])
```

Comment:
```
Headers: http.Header{
			"Custom-Header0": {"Custom-Value0", "Custom-Value00"},
			"Custom-Header1": {"Custom-Value1", "Custom-Value11"},
			"Custom-Header2": {"Custom-Value2", "Custom-Value22"},
		},
```

In case the headers contain more than one value, then with the current implementation, only the first value will be considered, which is wrong as it ignores the other values. 

Could you pls fix this behavior?

Reply:
Do we support multiple values as input in the Header ??

Reply:
Since as per the usecase `IBM-Req-Ctx-Token` only accepts single input from the user, it will throw error for multiple inputs however to support functionality we are allowing multiple inputs for futher usecase